.circleci/config.yml

```diff
@@ -35,6 +35,44 @@ jobs:
     steps:
       - checkout
 
+      - run:
+          name: Install Debian packaging dependencies and download wheels
+          command: |
+            mkdir ~/packaging && cd ~/packaging
+            git clone https://github.com/freedomofpress/securedrop-debian-packaging.git
+            cd securedrop-debian-packaging
+            make install-deps && make fetch-wheels
+            PKG_DIR=~/project make requirements
+
+      - run:
+          name: Ensure that build-requirements.txt and requirements.txt are in sync.
+          command: |
+            cd ~/project
+            # Return 1 if unstaged changes exist (after `make requirements` in the
+            # previous run step), else return 0.
+            git diff --quiet
+
+      - run:
+          name: Tag and make source tarball
+          command: |
+            cd ~/project
+            ./update_version.sh 1000.0  # Dummy version number, doesn't matter what we put here
+            python3 setup.py sdist
+
+      - run:
```

Comment:
want to make these run steps reusable by adding them to `common-steps` above as [YAML anchors](https://blog.daemonl.com/2016/02/yaml.html) and then referencing them here and in the below `build-buster` job? (that way we keep these CI configs DRY)

Reply:
I will make this change, I never knew about YAML anchors before.